tuf/ngclient/updater.py

```diff
@@ -417,10 +429,9 @@ def _preorder_depth_first_walk(
 
             # The metadata for 'role_name' must be downloaded/updated before
             # its targets, delegations, and child roles can be inspected.
-            self._load_targets(role_name, parent_role)
+            role_metadata = self._load_targets(role_name, parent_role)
```

Comment:
Maybe something like this:
```suggestion
            targets = self._load_targets(role_name, parent_role).signed
```
and then using `targets` in the rest of the code instead of `role_metadata.signed` would make the code lines a little shorter -- I realize it leads to using "targets.targets", but that's more an issue with the API...